engine/bindings/python/interface.cpp

```diff
@@ -312,7 +312,7 @@ static PyObject *query(PyObject *self, PyObject *args, PyObject *kwargs) {
   if (!PyArg_ParseTupleAndKeywords(
           args,
           kwargs,
-          "ssOOip",
+          "ssOOisp",
```

Comment:
What is this?

Reply:
https://github.com/epsilla-cloud/vectordb/pull/52 breaks the pybinding because the position argument for the new "filter" is not specified.

Reply:
Then why "ss00isp" instead of "filter"?

Reply:
this is the argument format. linke printf

"filter" is defined here: https://github.com/epsilla-cloud/vectordb/pull/53/files/0df66861a5cc4f2ee6000ab0998985c9e62bd12c..a708f909ab8b30bb13d94ec70d12bcb222a09cdd#diff-b62002569bb337121cefbf7f0874e4ea8e0f0f1ff9821cd91f1b9038990e409bL322

Reply:
and here: https://github.com/epsilla-cloud/vectordb/pull/53/files/0df66861a5cc4f2ee6000ab0998985c9e62bd12c..a708f909ab8b30bb13d94ec70d12bcb222a09cdd#diff-b62002569bb337121cefbf7f0874e4ea8e0f0f1ff9821cd91f1b9038990e409bR305